style.css

```diff
@@ -0,0 +1,675 @@
+  -moz-box-sizing: border-box;
+}
+
+body {
+  overflow-x: hidden;
+  background: url(assets/header-bg.jpg);
+  background-size: 100%;
+}
+
+p {
+  font-size: 20px;
+}
+
+p.small {
+  font-size: 16px;
+}
+
+.navbar {
+  width: 100%;
+  height: 70px;
+  display: flex;
+  flex-direction: row;
+  padding-top: 10px;
+  justify-content: space-between;
+  position: fixed;
+  background: rgba(242, 242, 242, 0.1);
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.navbar .logo-container {
+  display: none;
+}
+
+#toggle-icon {
+  display: block;
+}
+
+#toggle-icon div {
+  width: 35px;
+  height: 3px;
+  background-color: black;
+  margin: 6px 0;
+  transition: 0.4s;
+  align-self: flex-end;
+}
+
+nav .nav-menu {
+  display: none;
+  position: fixed;
+  top: 50px;
+  left: 0;
+  height: calc(100vh - 50px);
+  width: 100%;
+}
+
+.open .nav-menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  list-style-type: none;
+  background-color: rgba(211, 211, 211, 0.1);
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+  z-index: 1;
+  padding: 30px 30px;
+}
+
+.nav-menu li {
+  margin-bottom: 40px;
+  cursor: pointer;
+}
+
+.nav-menu li a {
+  font-family: "Lato", sans-serif;
+  font-size: 18px;
+}
+
+.open .bar1 {
+  -webkit-transform: rotate(-45deg) translate(-6px, 6px);
+  transform: rotate(-45deg) translate(-6px, 6px);
+}
+
+.open .bar2 {
+  opacity: 0;
+}
+
+.open .bar3 {
+  -webkit-transform: rotate(45deg) translate(-6px, -8px);
+  transform: rotate(45deg) translate(-6px, -8px);
+}
+
+.links-container ul {
+  display: none;
+}
+
+header {
+  padding-top: 50px;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+header .header-container {
+  display: flex;
+  flex-direction: column;
+  margin-top: 35px;
+}
+
+.header-container .welcome-heading {
+  color: #ec5242;
+  font-family: "Lato", sans-serif;
+  font-size: 17px;
+}
+
+.header-container .intro {
+  font-family: "cocogoose-trial", sans-serif;
+  text-transform: uppercase;
+  display: block;
+  margin-top: 30px;
+  color: rgba(236, 82, 66, 1);
+  -webkit-background-clip: text;
+  font-size: 40px;
+  max-width: 500px;
+  font-weight: normal;
+}
+
+.header-container .intro-paragraph {
+  border: 2px solid white;
+  background-color: #d3d3d3;
+  padding: 15px;
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  color: #4a4848;
+  margin-top: 30px;
+  max-width: 500px;
+}
+
+.header-container .date {
+  font-family: "Cocogoose-trial", sans-serif;
+  font-size: 30px;
+  color: #4a4848;
+  margin-top: 25px;
+}
+
+.main-program {
+  background: url(assets/mailing-bg.jpg);
+  padding-top: 25px;
+  padding-bottom: 25px;
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+}
+
+.main-program .main-program-heading {
+  color: white;
+  font-family: "Lato", sans-serif;
+  text-align: center;
+  margin-bottom: 12px;
+}
+
+.main-program-hr {
+  width: 35px;
+  align-self: center;
+  color: #ec5242;
+}
```

Comment:
- [ ] Kindly make sure to meet the color of the underline on the different titles. Please fix that to meet the provided Behance design.

  - ### YOUR DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/225587367-72f9fc5a-9bb0-4365-bc52-47879cf83ca9.png)

    ![image](https://user-images.githubusercontent.com/87197412/225587455-694f1c70-44dd-4a88-8719-10ee889485bc.png)

   ![image](https://user-images.githubusercontent.com/87197412/225587533-c73cda46-2aed-4724-b50c-fb13c35045b9.png)


  - ### FIGMA DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/225587615-79efa7a0-4649-4cca-a6a9-5745095fd7f5.png)

    ![image](https://user-images.githubusercontent.com/87197412/225587696-a2eea4f3-57bc-4b30-b06f-3085e3fb7567.png)

   ![image](https://user-images.githubusercontent.com/87197412/225587758-861f6e07-85cc-45eb-a1c2-59a997c66403.png)